### PR TITLE
Minor Larva bugfixes

### PR DESCRIPTION
**Pull request by nielsm5**
Turns multithreaded into opt-in.

Fixes:
image

### CODE REVIEW

File: larva/src/main/java/org/frankframework/larva/LarvaTool.java [hunk 1]
```diff
@@ -448,11 +448,11 @@ public void printHtmlForm(List<String> scenariosRootDirectories, List<String> sc
 		writeHtml("<span style=\"float: left; font-size: 10pt; width: 0px\">&nbsp; &nbsp; &nbsp;</span>", false);
 		writeHtml("<table style=\"float:left;height:50px\">", false);
 		writeHtml(TR_STARTING_TAG, false);
-		writeHtml("<td>Multi threaded (beta)</td>", false);
+		writeHtml("<td>Multi Threaded (experimental)</td>", false);
 		writeHtml(TR_CLOSING_TAG, false);
 		writeHtml(TR_STARTING_TAG, false);
 		writeHtml(TD_STARTING_TAG, false);
-		writeHtml("<input type=\"checkbox\" name=\"multithreaded\" value=\"true\"", false);
+		writeHtml("<input type=\"checkbox\" name=\"multithreaded\"", false);
```
Comment (tnleeuw) [inline larva/src/main/java/org/frankframework/larva/LarvaTool.java:455]: This is not consistent with other checkboxes, which all send the value `"true"` when checked.
Writing value `true` doesn't mean the checkbox is checked, that is done on line 457.

So I suggest reverting this and like 182, to be consistent with all other checkboxes in the file.

See for instance:
https://stackoverflow.com/questions/11424037/do-checkbox-inputs-only-post-data-if-theyre-checked

File: larva/src/main/java/org/frankframework/larva/LarvaTool.java [hunk 2]
```diff
@@ -179,8 +179,8 @@ public int runScenarios(IbisContext ibisContext, String paramLogLevel, String pa
 		if (paramAutoScroll == null && paramLogLevel != null) {
 			config.setAutoScroll(false);
 		}
-		if (paramMultithreaded == null && paramLogLevel != null) {
-			config.setMultiThreaded(false);
+		if (("on".equals(paramMultithreaded) || Boolean.parseBoolean(paramMultithreaded)) && paramLogLevel != null) {
+			config.setMultiThreaded(true);
```
Comment (tnleeuw) [inline larva/src/main/java/org/frankframework/larva/LarvaTool.java:183]: I think this check should not depend on `paramLogLevel`. I also think we can revert the `"on"` check to a `null` check, and set the checkbox directly from `paramMultiThreaded != null`, 

Something like this:
```suggestion
		config.setMultiThreaded(paramMultithreaded != null);
```

File: larva/src/main/java/org/frankframework/larva/TestConfig.java [hunk 3]
```diff
@@ -21,24 +21,22 @@
...
+	private @Getter @Setter LarvaLogLevel logLevel = LarvaLogLevel.WRONG_PIPELINE_MESSAGES;
 
-	private boolean autoScroll = true;
-	private boolean useHtmlBuffer = false;
-	private boolean useLogBuffer = true;
-	private boolean multiThreaded = true;
+	private @Getter @Setter boolean autoScroll = true;
+	private @Getter @Setter boolean useHtmlBuffer = false;
+	private @Getter @Setter boolean useLogBuffer = true;
+	private @Getter @Setter boolean multiThreaded = false;
```
Comment (tnleeuw) [inline larva/src/main/java/org/frankframework/larva/TestConfig.java:36]: This was the real problem.

**Review by tnleeuw — CHANGES_REQUESTED** [review]

File: larva/src/main/java/org/frankframework/larva/TestConfig.java (hunk 3 above)
Comment (nielsm5) [reply larva/src/main/java/org/frankframework/larva/TestConfig.java:36]: One of 4 problems.

File: larva/src/main/java/org/frankframework/larva/LarvaTool.java (hunk 2 above)
Comment (nielsm5) [reply larva/src/main/java/org/frankframework/larva/LarvaTool.java:183]: Due to the poor design of this method, you need to be able to change a value without larva running all scenario's.
Not why why this was like this, it's not been changed, and I do not intend to do so.

File: larva/src/main/java/org/frankframework/larva/LarvaTool.java (hunk 1 above)
Comment (nielsm5) [reply larva/src/main/java/org/frankframework/larva/LarvaTool.java:455]: What does this achieve?

File: larva/src/main/java/org/frankframework/larva/LarvaTool.java [hunk 4]
```diff
@@ -670,7 +670,7 @@ public void wrongPipelineMessage(String stepDisplayName, String message, String
 		writeLog("<pre id='" + diffBoxId + "' class='diffBox'></pre>", method, false);
 		writeLog("</div>", method, false);
 
-		if (config.getLogLevel().shouldLog(LarvaLogLevel.SCENARIO_PASSED_FAILED)) {
+		if (config.getLogLevel() == LarvaLogLevel.SCENARIO_PASSED_FAILED) {
```
Comment (evandongen) [inline larva/src/main/java/org/frankframework/larva/LarvaTool.java:673]: Hiermee vergelijk je de reference en niet per se de inhoud.
```suggestion
		if (LarvaLogLevel.SCENARIO_PASSED_FAILED.equals(config.getLogLevel())) {
```

**Review by evandongen — CHANGES_REQUESTED** [review]

File: larva/src/main/java/org/frankframework/larva/LarvaTool.java (hunk 4 above)
Comment (nielsm5) [reply larva/src/main/java/org/frankframework/larva/LarvaTool.java:673]: Dit is een enum :)

File: larva/src/main/java/org/frankframework/larva/LarvaTool.java (hunk 1 above)
Comment (tnleeuw) [reply larva/src/main/java/org/frankframework/larva/LarvaTool.java:455]: Consistency with rest of the code, that writes checkboxes with `value="true"` also when not checked.

File: larva/src/main/java/org/frankframework/larva/LarvaTool.java (hunk 4 above)
Comment (evandongen) [reply larva/src/main/java/org/frankframework/larva/LarvaTool.java:673]: Oeps

**Review by evandongen — APPROVED** [review]

**Review by tnleeuw — APPROVED** [review]

**Pull request merged**